matter_server/client/client.py

```diff
@@ -231,6 +231,28 @@ async def send_device_command(
             interaction_timeout_ms=interaction_timeout_ms,
         )
 
+    async def write_attribute(
```

Comment:
What features need to write attributes? Why doesn't all features that does a change have a device command?

Reply:
commands that are not predefined in the clusters, manufacturer specific data.
For example setting the mode of a thermostat is not defined as cluster command but thermostats can accept it with writing an attribute

Reply:
see chapter 8.9.2.4 of the specs btw